### PR TITLE
add Busuu, Mine. update Sololearn

### PR DESCRIPTION
**Pull request by Zumux**
- Add Busuu as easy
- Add Mine as easy
- Update Sololearn to easy

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -2102,6 +2102,16 @@
         ]
     },
 
+    {
+        "name": "Busuu",
+        "url": "https://bussu.com",
```
Comment (tupaschoal) [inline _data/sites.json:2107]:
```suggestion
        "url": "https://busuu.com",
```

File: _data/sites.json [hunk 2]
```diff
@@ -2102,6 +2102,16 @@
...
+        "difficulty": "easy",
+        "notes": "Click the 'Delete my account' button at the bottom of the page, then confirm you'd like to delete your account",
```
Comment (tupaschoal) [inline _data/sites.json:2109]: Is there a direct link to the profile/settings page that contains this button?

**Comment by Zumux** [comment]
Fixed.

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**